packages/governance/src/contractHelper.js

```diff
@@ -152,7 +152,8 @@ const facetHelpers = (zcf, paramManager) => {
    *
    * @see {makeDurableGovernorFacet}
    *
-   * @param {{ [methodName: string]: (context?: unknown, ...rest: unknown[]) => unknown}} limitedCreatorFacet
+   * @template {{ [methodName: string]: (context?: unknown, ...rest: unknown[]) => unknown}} LCF
+   * @param {LCF} limitedCreatorFacet
```

Comment:
There seems to be an endless amount of refinement to do for the types for governance.

I wonder if really complex types is like really complex documentation: it suggests searching for a simpler design.

Reply:
Indeed. The design we shipped with was somewhat rushed. Making it durable is also rushed (and incomplete). When making it durable we could simplify, or do that in a next phase.